SpatialGDK/Source/SpatialGDK/SpatialGDK.Build.cs

```diff
@@ -48,40 +48,44 @@ public SpatialGDK(ReadOnlyTargetRules Target) : base(Target)
         string SharedLibSuffix = "";
         bool bAddDelayLoad = false;
 
-        switch (Target.Platform)
+        if (Target.Platform == UnrealTargetPlatform.Win32 || Target.Platform == UnrealTargetPlatform.Win64)
```

Comment:
Why the change from switch to if?

Reply:
Target.Platform was changed from a string (? I think) to a class or another type that is no longer able to be switched on.

Reply:
🤢